packages/kbn-pm/src/commands/bootstrap.ts

```diff
@@ -58,11 +63,23 @@ export const BootstrapCommand: ICommand = {
     // That way non bazel projects could depend on bazel projects but not the other way around
     // That is only intended during the migration process while non Bazel projects are not removed at all.
     //
+
     if (forceInstall) {
+      const forceInstallStartTime = Date.now();
       await runBazel(['run', '@nodejs//:yarn'], runOffline);
+      timings.push({
+        id: 'yarn install',
```

Comment:
I'm not in love with calling this just "yarn install", but I'm struggling to come up with a better id...

Reply:
`install dependencies`?

Reply:
I'm more concerned that it only accounts for initial installs and when people run with `--force-install`

Reply:
I's suggest to report all options people pass in to commands for furhter investigation

Reply:
Calling it `force install dependencies` for now just so we can identify what it is. I don't think the name for this really matters much as it should eventually go away entirely.